utility/mm2_update-master-directory-list

```diff
@@ -52,21 +53,32 @@ class MasterFilter(logging.Filter):
         record.category = cname
         return True
 
-def make_file_details_from_checksums(session, diskpath, relativeDName, D):
+def make_file_details_from_checksums(session, config, D, files=None):
     def _parse_checksum_file(relativeDName, checksumlen):
         r = {}
         try:
-            f = open(os.path.join(diskpath, relativeDName),  'r')
+            f = open(os.path.join(config.get('UMDL_PREFIX', ''), relativeDName),  'r')
```

Comment:
Yup, that will automatically close the file when exiting the block of code (even if an exception occurs)